README.md

```diff
@@ -1 +1,97 @@
 # deis-cleanup
+
```

Comment:
Maybe write a sentence why is it needed and why we had to write our own solution